accounting/receivables/customer_payments/check.rst

```diff
@@ -1,3 +1,148 @@
-==================================
-How to record a payment by checks?
-==================================
+============================================
+How to register customer payments by checks
+============================================
+
+There are two ways to handle payments received by checks. Odoo support
+both approaches so that you can use the one that better fits your
+habits.
+
+1. Undeposited Funds: 
+    once you receive the check, you record a payment
+    by check on the invoice. (using a Check journal and posted on the
+    Undeposited Fund account) Then, once the check arrives in your
+    bank account, move money from Undeposited Funds to your bank
+    account.
+
+2. One journal entry only: 
+    once your receive the check, you record a
+    payment on your bank, paid by check, without going through the
+    Undeposited Funds. Once you process your bank statement, you do
+    the matching with your bank feed and the check payment, without
+    creating a dedicated journal entry.
+
+We recommend the first approach as it is more accurate (your bank
+account balance is accurate, taking into accounts checks that have not
+been cashed yet). Both approaches require the same effort.
+
+Even if the first method is cleaner, Odoo support the second approach
+because some accountants are used to it (quickbooks and peachtree
+users).
+
+Note: You may have a look at the *Deposit Ticket feature* if you deposit
+several checks to your bank accounts in batch.
+
+Option 1: Undeposited Funds
+===========================
+
+Configuration
+-------------
+
+-  Create a journal “Checks”
+
+-  Set “Undeposited Checks” as a defaut credit/debit account
```

Comment:
I usually put all field names and options in bold instead of using quotation marks (but I guess it's also acceptable) but anyway wrong quotation marks then